test/functional/executor.py

```diff
@@ -62,8 +62,10 @@ def run(self):
                 print("TEST: ", test.get_name())
                 print("DESCRIPTION: ", test.get_description())
                 print("RUNNING: ", *cmd)
-            subprocess.run(cmd, stdout=subprocess.DEVNULL,
-                           stderr=subprocess.DEVNULL)
+                print(subprocess.getoutput(cmd))
```

Comment:
This function is a legacy, python 2.x compatibility. I don't think we should use it. `run` returns `subprocess.CompletedProcess` which allows to get `stdout` if it was captured (it can be captured using for example `stdout=subprocess.PIPE, stderr=subprocess.STDOUT`). Please note that by default `stdout` will be a byte array not a string:
```
>>> x = subprocess.run(['ls', '-l', '/dev/null'], stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
>>> print(x.stdout)
b'crw-rw-rw- 1 root root 1, 3 01-25 19:40 /dev/null\n'
>>> print(x.stdout.decode(sys.stdout.encoding))
crw-rw-rw- 1 root root 1, 3 01-25 19:40 /dev/null

>>> 
```
Also, here and in lines below we are not checking the return status of the command. (`returncode` from `subprocess.CompletedProcess`) In case the code is different that 0 we should mark test as failed for unknown reason.
Please refer to https://docs.python.org/3/library/subprocess.html
Please also note that currently our cmake is only checking if python *3.x* is available. It should check for *3.5* for `run` function.